docs/metrics.md

```diff
@@ -0,0 +1,136 @@
+
+### Custom Dimensions
+- `cd1` - the method by which the user initiated an upload. One of `drag`, `click`
+- `cd2` - the reason that the file transfer stopped. One of `completed`, `errored`, `cancelled`.
+- `cd3` - the destination of a link click. One of `experiment-page`, `download-firefox`, `twitter`, `github`, `cookies`, `terms`, `privacy`, `about`, `legal`, `mozilla`.
+- `cd4` - from where the URL for a file was copied. One of `finished-screen`, `file-list`
+- `cd5` - the referring location. One of `completed-download`, `errored-download`, `cancelled-download`, `completed-upload`, `errored-upload`, `cancelled-upload`, `testpilot`, `external`.
+- `cd6` - the location from which the user copied the URL to an upload file. One of `success-screen`, `upload-list`.
+- `cd7` - identifying information about an error. Exclude if there is no error involved. **TODO:** enumerate a list of possibilities.
+
+### Events
+
+_NB:_ due to how files are being tracked, there are no events indicating file expiry. This carries some risk: most notably, we can only derive expiration rates by looking at download rates, which is prone to skew if there are problems in data collection.
+
+#### `upload-started`
+Triggered whenever a user begins uploading a file. Includes:
+
+- `ec` - `sender`
+- `ea` - `upload-started`
+- `cm1`
+- `cm5`
+- `cm6`
+- `cm7`
+- `cd1`
+- `cd5`
+
+#### `upload-stopped`
+Triggered whenever a user stops uploading a file. Includes:
+
+- `ec` - `sender`
+- `ea` - `upload-stopped`
+- `cm1`
+- `cm2`
+- `cm3`
+- `cm5`
+- `cm6`
+- `cm7`
+- `cd1`
+- `cd2`
+- `cd7`
+
+#### `download-started`
+Triggered whenever a user begins downloading a file. Includes:
+
+- `ec` - `recipient`
+- `ea` - `download-started`
+- `cm1`
+- `cm4`
+- `cm5`
+- `cm6`
+- `cm7`
+
+#### `download-stopped`
+Triggered whenever a user stops downloading a file.
+
+- `ec` - `recipient`
+- `ea` - `download-stopped`
+- `cm1`
+- `cm2` (if possible and applicable)
+- `cm3` (if possible and applicable)
+- `cm5`
+- `cm6`
+- `cm7`
+- `cd2`
+- `cd7`
+
+#### `exited`
+Fired whenever a user follows a link external to Send.
+
+- `ec` - `recipient`, `sender`, or `other`, as applicable.
+- `ea` - `exited`
+- `cd3`
+
+#### `upload-deleted`
+Fired whenever a user deletes a file they’ve uploaded.
+
+- `ec` - `sender`
+- `ea` - `upload-deleted`
+- `cm1`
+- `cm2`
+- `cm3`
+- `cm4`
+- `cm5`
+- `cm6`
+- `cm7`
+- `cd1`
+
+#### `copied`
+Fired whenever a user copies the URL of an upload file.
+
+- `ec` - `sender`
+- `ea` - `copied`
+- `cd6`
+
+#### `restarted`
+Fired whenever the user interrupts any part of funnel to return to the start of it (e.g. with a “send another file” or “send your own files” link).
+
+- `ec` - `recipient`, `sender`, or `other`, as applicable.
+- `ea` - `restarted`
+- `cd2`
+
+#### `unsupported`
+Fired whenever a user is presented a message saying that their browser is unsupported due to missing crypto APIs.
+
+- `ec` - `sender`
+- `ea` - `unsupported`
+- `cd7`
```

Comment:
General thought... I'm not very clever, and metrics is confusing, but by the time I've scrolled down to here, I've already forgotten what a `cd7` is.

Not sure how ludicrous it would be to wrap some of these in an `<code title="cd 7 -- identifying information about an error. Exclude if there is no error involved.">cd7</code>` so you can just hover over the custom dimension to get a description. Markdown+HTML are hard. It appears GFM doesn't support ye olde `<abbr>` tag.

**Example** (hover for tooltip): 

<code title="cd 7 -- identifying information about an error. Exclude if there is no error involved.">cd7</code>